db/seeds.rb

```diff
@@ -52,17 +56,17 @@ def create_orders
 
    def beer
      [
-      ["Big Tasty", true, "Poignant and personal", 4 ],
-      ["Imperial Stout", true, "Generic Imperial stout", 4 ],
-      ["KB Special", true, "Description", 4 ],
-      ["Tracy Spacy", true, "Description", 4 ],
-      ["Minnie Winny", true, "Description", 4 ],
-      ["Pink Pilsner", true, "Description", 4 ],
-      ["Kreamy Kolsch", true, "Description", 4 ],
-      ["Hoppy Hefferveisen", true, "Description", 4 ],
-      ["Brown Blackout", true, "Description", 4 ],
-      ["Black Brownout", false, "Description", 3 ],
-      ["Duff", true, "Tasty and true", 6 ]
+      ["Big Tasty", true, "Poignant and personal", 400 ],
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.<br>Space inside square brackets detected.